README.md

```diff
@@ -12,7 +12,7 @@ This repo is a holding area for recipes destined for a conda-forge feedstock rep
 ## Getting started
 
 1. Fork this repository.
-2. Make a new folder in `recipes` for your package. Look at the example recipe and our [FAQ](https://github.com/conda-forge/staged-recipes/tree/readme_faq#faq) for help.
```

Comment:
This change is fine, but I would like it to be a separate PR from the recipe addition as it is good git hygiene.